src/content/peerconnection/bandwidth/js/main.js

```diff
@@ -187,7 +187,7 @@ bandwidthSelector.onchange = function() {
   .then(function() {
     var desc = pc1.remoteDescription;
     desc.sdp = desc.sdp.replace(/b=AS:(.*)\r\n/, 'b=AS:' + bandwidth + '\r\n');
-    trace('Applying bandwidth change to setRemoteDescription: ' + desc);
+    trace('Applying bandwidth change to setRemoteDescription: ' + desc.sdp);
```

Comment:
You probably want the SDP on a new line, since it's multi line.

`trace('Applying bandwidth change to setRemoteDescription:\n' + desc.sdp);`